app/lib/presentation/screens/home.dart

```diff
@@ -71,57 +71,82 @@ class Home extends StatelessWidget {
                     itemBuilder: (context, index) {
                       return Padding(
                         padding: const EdgeInsets.all(8.0),
-                        child: Card(
-                          child: ListTile(
-                            title: Text(
-                              state.accounts[index].accountName,
-                              style:
-                                  const TextStyle(fontWeight: FontWeight.bold),
-                            ),
-                            subtitle: Text(
-                              '${state.accounts[index].accountNumber.toString()} - ${state.accounts[index].bankName}',
-                            ),
-                            trailing: Row(
-                              mainAxisSize: MainAxisSize.min,
-                              children: [
-                                GestureDetector(
-                                  onTap: () {
-                                    // copy data to clipboard
-                                    Clipboard.setData(
-                                      ClipboardData(
-                                        text: state
-                                            .accounts[index].accountNumber
-                                            .toString(),
-                                      ),
-                                    );
-                                    ScaffoldMessenger.of(context).showSnackBar(
-                                      const SnackBar(
-                                        content: Text("Copied to clipboard"),
-                                      ),
-                                    );
-                                  },
-                                  child: const Icon(
-                                    Icons.copy,
-                                    color: Colors.deepPurple,
-                                  ),
+                        child: Dismissible(
+                          confirmDismiss: (direction) => showDialog<bool>(
+                            context: context,
+                            builder: (BuildContext ctx) => AlertDialog(
+                              title: const Text('Alert'),
```

Comment:
Title should be "Delete Account" ideally with warning emoji/icon and the background/color of the dialog can be made to some shade of red to indicate it's a dangerous action.

Reply:
making the dialog background red would be nice but I think it wouldn't sync well with the backdrop, alternatively the title can be red and the warning icon too